### ISSUE TITLE
Allow more chars in extra properties for PropertySpelling

### ISSUE THREAD

**Issue by franz-josef-kaiser** [brigade/scss-lint#550]
The [`PropertySpelling`](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertyspelling) property does not allow 

```yml
  PropertySpelling:
    extra_properties:
      - *zoom
```

It "works" when put in quotes (both single and double), but then does not work anymore.

**Comment by sds** [comment]
Hey @franz-josef-kaiser,

This is a feature YAML: the `*` is a special character that gets interpreted as an alias. You'll receive a parse error in this case.

The only workaround is to use quotes. Hope that helps!

**Comment by franz-josef-kaiser** [comment]
@sds ok, that's something I understand. Problem still is: The parser does not work anymore when put in quotes. The statement simply gets ignored (hence I put "works" in quotes). Please reopen as this isn't fixed.

**Comment by sds** [comment]
I'm sorry @franz-josef-kaiser, I don't think I understand your issue.

When I have the following `.scss-lint.yml` configuration:

```yaml
linters:
  PropertySpelling:
    extra_properties:
      - '*zoom'
```

...and I run it against the following SCSS:

```scss
p {
  *zoom: 1px;
}
```

...no errors are reported, and thus everything works as intended. Can you provide a clear example of the output/behavior you are expecting?

**Comment by franz-josef-kaiser** [comment]
Well, we run it again (and re-ran it on Codeship) and got the following `exit` code:

```
dev/assets/sass_new/global/_mixins.scss
57: [W] PropertySpelling:  Unknown property *zoom
```

The used property is

```
*zoom: 1;
```

That's from our `diff`:

```
   PropertySpelling:
     extra_properties:
-      - "*zoom"
+      - '*zoom'
```

**Comment by sds** [comment]
What other configuration options are in your `.scss-lint.yml` file? Are those other configuration options working as expected? Are you sure `scss-lint` is using the correct configuration file?

`scss-lint` 0.40.1 is working as expected for me locally, which tells me this is a configuration issue on your side. If you run `scss-lint dev/assets/sass_new/global/_mixins.scss --config=path/to/your/scss-lint.yml` directly do you still see the error?
